server/graphql/reports/src/lib.rs

```diff
@@ -38,8 +44,15 @@ impl ReportQueries {
             desc = "The data id that should be used for the report, e.g. the invoice id when printing an invoice"
         )]
         data_id: String,
+        format: Option<PrintFormat>,
     ) -> Result<PrintReportResponse> {
-        print_report(ctx, store_id, report_id, data_id).await
+        let report_format = match format {
+            Some(PrintFormat::Html) => Some(service::report::report_service::PrintFormat::Html),
```

Comment:
I wanted to use an enum here and in the report_service but they didn't like it when I tried to use the same one. So I made two and mapped.. bad?